### DISCUSSION TITLE
Client with No Tags

### DISCUSSION THREAD

**Discussion by jithinav** [madrecha/jamku#668] [QA] [answered]
In Client List page > Table Mode

The Tags column filter doesn't have "blank" in the list.

![image](https://user-images.githubusercontent.com/78638740/124942337-2a20b680-e029-11eb-9efe-e1a98beb99fe.png)

**Comment by adarshmadrecha** [comment, marked as answer]
Instead of Filtering, simply sort the Client List basis Tags.
This will bring all the clients with NO tags to the top.
